errno/errname.go

```diff
@@ -946,6 +946,10 @@ var MySQLErrName = map[uint16]string{
 	ErrJSONValueOutOfRangeForFuncIndex:                       "Out of range JSON value for CAST for functional index '%s'",
 	ErrFunctionalIndexDataIsTooLong:                          "Data too long for functional index '%s'",
 	ErrFunctionalIndexNotApplicable:                          "Cannot use functional index '%s' due to type or collation conversion",
+	ErrUnsupportedColumnCheck:                                "Column check is not supported",
+	ErrUnsupportedAlterCheck:                                 "ALTER CHECK is not supported",
+	ErrUnsupportedDropCheck:                                  "DROP CHECK is not supported",
+	ErrUnsupportedConstraintCheck:                            "ADD CONTRAINT CHECK is not supported",
```

Comment:
I think only one error is necessary, WDYT @chanme 

```go
ErrUnsupportedConstraintCheck:                            "%s is not supported",
```

Reply:
You're right, that seems more concise, I will change the errname

Reply:
@tangenta I've changed the errname and corresponding unit tests.(BTW, It seems the unit test fails because the parser still generate the warning info, I think I should wait for the #888 to be merged first.)

Reply:
You can try 
```
go mod edit -replace github.com/pingcap/parser=github.com/chanme/parser@5ee466577ad4
```
, after the tests passed, we can merge the parser PR.

Reply:
I've already added the replace in go.mod to do the unit tests. In my local machine it passes the test cases, Do I need to commit something else to pass the "idc-jenkins-ci-tidb/unit-test" ?
(replace github.com/pingcap/parser => github.com/chanme/parser v0.0.0-20200607021413-839e39b9953a)

Reply:
> In my local machine it passes the test cases
Ok, we can merge the parser PR after getting 2 LGTMs.